src/join.sol

```diff
@@ -427,3 +427,113 @@ contract AuthGemJoin is LibNote {
         require(gem.transfer(usr, wad), "AuthGemJoin/failed-transfer");
     }
 }
+
+interface GemLike7 {
+    // matches ERC20 spec
+    function decimals() external view returns (uint);
+    // matches ERC20 spec
```

Comment:
This is misleading, it's not a match because it doesn't return `bool`

Reply:
removed the comments